src/Selenium2Driver.php

```diff
@@ -142,7 +142,7 @@ public function setDesiredCapabilities($desiredCapabilities = null)
                 $desiredCapabilities['chrome.'.$capability] = $value;
             }
 
-            $desiredCapabilities['chromeOptions'] = $chromeOptions;
+            $desiredCapabilities['goog:chromeOptions'] = $chromeOptions;
```

Comment:
Please rebase, because this change was already made in #312.

Reply:
Rebase done !